news/PR-0658.rst

```diff
@@ -0,0 +1,15 @@
+**Added:** None
+
+**Changed:** None
```

Comment:
Not "None"

Just an FYI but there needs to be a blank line between the `**Changed:**` line and the list of changes or else the list won't get formatted correctly

Reply:
See here https://github.com/ngranda/DAGMC/blob/update_version/news/PR-0658.rst